yb-voyager/cmd/importData.go

```diff
@@ -309,8 +309,8 @@ func importData() {
 
 func createVoyagerSchemaOnTarget(connPool *tgtdb.ConnectionPool) error {
 	cmds := []string{
-		"CREATE SCHEMA IF NOT EXISTS ybvoyager",
-		`CREATE TABLE IF NOT EXISTS ybvoyager.batches (
+		"CREATE SCHEMA IF NOT EXISTS ybvoyager_metadata",
+		`CREATE TABLE IF NOT EXISTS ybvoyager_metadata.batches (
 			schema_name VARCHAR(250),
 			file_name VARCHAR(250),
 			rows_imported BIGINT,
```

Comment:
Nit:
The error message can also be edited on line number `327` to:
`return fmt.Errorf("create ybvoyager_metadata schema on target: %w", err)`